quirks/web-browser-extension-distribution-information.json

```diff
@@ -352,6 +352,25 @@
             "supported_store_identifiers": [
                 5
             ]
+        },
+        {
+            "long_name": "Orion",
```

Comment:
maybe the long name should be “Orion Browser by Kagi”?

Reply:
Sure! Wasn't sure if this was a formal thing found in the application bundle or a convenience thing for this project. I'll update